adr/0022-integration-quality-scale.md

```diff
@@ -0,0 +1,193 @@
+
+🥉 Bronze: This is the baseline standard and requirement for all new integrations added to Home Assistant. The integration can be configured via the UI, and there is documentation on how to set it up.
+
+🥈 Silver: Enhanced reliability and robustness of the integration, ensuring a solid runtime experience. The integration handles errors correctly, including authentication errors and offline devices. The documentation provides more information on what the integration exactly provides. The integration has one or more code owners who help to maintain the integration and ensure this runtime experience lasts.
+
+🥇 Gold: The gold standard in delivering the best possible user experience with advanced features and comprehensive support. They are fully featured, user-friendly, and accessible to a wider audience. It is fully translated, capable of discovering their devices, providing device updates through Home Assistant, and has extensive documentation with examples, use cases, and troubleshooting. They provide means for debugging issues and can be fully adjusted through the UI.
+
+🏆 Platinum: The epitome of quality within Home Assistant. Not only does this integration provide an outstanding user experience, but it also adheres to the highest technical standards with optimized performance, efficiency, and code quality.
+
+Besides the four scaled tiers above, we also have four additional special tiers:
+
+❓ No score: An integration that is available via the UI but has not been evaluated against the integration quality scale or does not meet the minimum requirements for scoring.
+
+🏠 Internal: This is an integration used internally by Home Assistant to provide basic components and building blocks for the core program of Home Assistant or for other integrations to built on. The Home Assistant project maintains these.
+
+💾 Legacy: An older integration that has not yet transitioned to the Home Assistant user interface and lacks modern (nowadays considered normal) features.
+
+📦 Custom: The Home Assistant project does not create, maintain, audit, review, or support custom integrations; therefore, they cannot be graded and placed on the integration quality scale.
+
+## Decision
+
+### Scaled tiers
+
+Home Assistant integration quality scale has four differentiated tiers to which integration can be graded: 🥉 Bronze, 🥈 Silver, 🥇 Gold, and 🏆 Platinum.
+
+Each tier builds on the previous tier; for example, “Silver” has everything required by “Bronze”, and “Gold” has everything both “Silver” and “Bronze” require.
+
+While some integrations may not be graded in the top tier, they might still implement parts of other (higher grade) tiers.
+
+#### 🥉 Bronze
+The bronze tier is the baseline standard and requirement for all new integrations. It meets the minimum requirements in code quality, functionality, and user experience. It complies with the fundamental expectations and provides a reliable foundation for users to interact with their devices and services.
```

Comment:
We don't need to repeat the description of the tiers again. Just reference the above text, then simply list the named rules required for each tier.